core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -1436,6 +1436,12 @@ public void onReceive(Context context, Intent intent) {
 
         @Override
         public void onReceive(Context context, Intent intent) {
+            if (isInitialStickyBroadcast ()) {
```

Comment:
no new imports, doesn't seem to be an existing method - where is this implemented?

Reply:
https://developer.android.com/reference/android/content/BroadcastReceiver.html#isInitialStickyBroadcast()

> Returns true if the receiver is currently processing the initial value of a sticky broadcast -- that is, the value that was last broadcast and is currently held in the sticky cache, so this is not directly the result of a broadcast right now.